Example/android/app/build.gradle

```diff
@@ -85,7 +85,6 @@ project.ext.react = [
     nodeExecutableAndArgs:  ["/usr/local/bin/node"]
 ]
 
-apply from: "../../node_modules/react-native/react.gradle"
```

Comment:
what did this contain? are they moved in this file?

Reply:
The path has been changed as specified in this doc https://react-native-community.github.io/upgrade-helper/?from=0.71.8&to=0.73.5

Reply:
i did not see in doc. Test once and resolve if this is okay.

Reply:
This line is no longer needed as this file is removed in this commit 
https://github.com/facebook/react-native/commit/d4a9bdc40e06bdda9565cc43ea5af5a13ff6f1cf